packages/babel-plugin-minify-numeric-literals/src/index.js

```diff
@@ -25,9 +26,13 @@ module.exports = function() {
             .replace(/e0/, "");
         }
 
-        const replacement =
+        let replacement =
           normal.length > exponential.length ? exponential : normal;
 
+        if (path.node.value > 0 && Number.isInteger(path.node.value))
```

Comment:
small micro optimisation we can do is to compute the hex only for positive integers. Move this block
```js
const hexadecimal = `0x${path.node.value.toString(16)}`;
```
inside the if statement.

Reply:
Thanks, I moved it int 9fe2436. I think this can be made to work for the same integers with opposite signs but haven't verified that yet.